packages/xo-web/src/common/intl/messages.js

```diff
@@ -774,15 +774,15 @@ const messages = {
   // ----- Pool actions ------
   addSrLabel: 'Add SR',
   addVmLabel: 'Add VM',
-  addHostLabel: 'Add Host',
+  addHostsLabel: 'Add Hosts',
```

Comment:
I think we can keep this message for the sake of not invalidating all its translations.